### DISCUSSION TITLE
how to add local images and access them into the project

### DISCUSSION THREAD

**Discussion by SuryodayYadav** [egovernments/Digit-Core#105] [Local Governance]
here i can see the url - https://cdn.jsdelivr.net/npm/@egovernments/digit-ui-css@1.0.7/img/m_seva_white_logo.png
if i want to use my local image then i should put these into img folder, but after putting any image, that image is not visible on the web? 
https://cdn.jsdelivr.net/npm/@egovernments/digit-ui-css@1.0.7/img/onlineService.png , though this image is in the same folder of img.
do i need to upload it somewhere first before accessing them? 
please guide me on this?

**Comment by jagankumar-egov** [comment]
An S3 Bucket with Public visibility has to be created to store all the assets 
Steps to creating an S3 bucket is mentioned here 
https://core.digit.org/guides/developer-guide/ui-developer-guide/ui-configuration-devops#configure-aws-s3-bucket

Steps to upload the image into any s3 bucket 
https://docs.aws.amazon.com/redshift/latest/dg/tutorial-loading-data-upload-files.html

So Once it is uploaded the same can be used locally and in deployments.

Note if you don't have s3 bucket then, it can be stored inside the CSS module, and refer following doc for updating CSS
https://core.digit.org/guides/developer-guide/ui-developer-guide/customisation/css-customisation
